adapters/amqp-vertx/src/main/java/org/eclipse/hono/adapter/amqp/impl/AmqpContext.java

```diff
@@ -181,4 +183,25 @@ Sample getTimer() {
         return timer;
     }
 
+    /**
+     * Gets the value for the <em>sampling.priority</em> span tag to be used for OpenTracing spans created in connection
+     * with this AmqpContext.
+     *
+     * @return An <em>OptionalInt</em> containing the value for the <em>sampling.priority</em> span tag or an empty
+     *         <em>OptionalInt</em> if no priority should be set.
+     */
+    public OptionalInt getTraceSamplingPriority() {
```

Comment:
IMHO all these accessors should be final as there is no obvious reason for subclasses to override them, is there?

Reply:
Right, I've changed that.